### PR TITLE
Add proper whitelabeled templates

### LINKED ISSUE #744: Add proper PDF templates for Nürnberg and Bayern
**Is your feature request related to a problem? Please describe.**
We switched from manually typesetting icons and instructions in the PDF to using a template.
This task is concerned with creating two .pdf files which will be used when generating PDFs.

**Describe the solution you'd like**
See below how such a template looks like:

pdf_template

Based on this template two .pdf files should be created for Nürnberg and Bayern. Ideally the PDFs are created using Inkscape because then we developers can make adjustments. Any other tool is fine though.

The template uses mm units for defining the content areas. The mm can be checked when inspecting the following file using inkscape: https://user-images.githubusercontent.com/905221/214042678-14bb9987-3fa1-44b6-ae1e-ec7050011b13.svg

For example the top QR Code has the following spacing: X=105mm, Y=70mm, Width=90mm, Height=90mm

**For Bayern ignore the two content areas at the bottom.**

Here is an sketch of the information which should be on the PDF for Nürnberg. Note: This is just a quick preview crafted by Daniel. You do not need need to make the PDF look similar! Just the information must be preserved!

nbg-pass-idee

* The border of the PDF should be about 1cm, not 2cm! Most printers can print a 1cm border!
* Remove the name after the "Guten Tag". We do not want to change this for every print.

A few more assets useful for this task:

* Google and Apple Badges:
apple-app-store-badge
google-play-store-badge

Add the following line at the bottom of the file: Google Play und das Google Play-Logo sind Marken von Google LLC.

* Nürnberg Assets:
pdf-logo
pdf-static-card-background
pdf-static-card-logo

```
    issuer: 'Stadt Nürnberg',
    appName: 'Sozialpass',
    appDownloadLink: 'https://download.nuernberg.sozialpass.app',
    greeting: (fullName: string) => [`Guten Tag, ${fullName}`, 'Ihr digitaler Nürnberg-Pass ist da!'],
    disclaimer: [
      'Bitte beachten Sie, dass der Nürnberg-Pass nur in Verbindung mit einem amtlichen und gültigen Lichtbildausweis gültig ist.',
    ],
```

* Bayern Assets:
pdf-logo
-b91b-87d6035c9c06.png)

```
    issuer: 'Bayerische Staatsministerium für Arbeit und Soziales, Familie und Integration',
    appName: 'Ehrenamtskarte Bayern',
    appDownloadLink: 'https://download.bayern.ehrenamtskarte.app',
    greeting: (fullName: string) => [`Guten Tag, ${fullName}`, 'Ihre digitale Eherenamtskarte ist da!'],
    disclaimer: [
      'Bitte beachten Sie, dass die Ehrenamtskarte nur in Verbindung mit einem amtlichen und gültigen Lichtbildausweis gültig ist.',
    ],
```

**Describe alternatives you've considered**
Typesetting the document using jsPDF did not work.

**Additional context**
None

### PR DESCRIPTION
**Pull request by maxammann**
Thanks to Antonia!

fixes #744

### CODE REVIEW

**Comment by maxammann** [comment]
grafik

File: administration/src/project-configs/nuernberg/config.ts [hunk 1]
```diff
@@ -16,6 +18,17 @@ const config: ProjectConfig = {
...
+    infoToDetails: (info, _region) => {
+      const expirationDay = info.expirationDay
+      if (!expirationDay) {
+        throw new Error('expirationDay must be defined for Nürnberg')
+      }
+
+      return `Name: ${info.fullName}
+Passnummer: ${info.extensions?.extensionNuernbergPassNumber?.passNumber}
+Geburtsdatum: ${format(daysSinceEpochToDate(info.extensions?.extensionBirthday?.birthday ?? 0), 'dd.MM.yyyy')}
+Gültigkeit: ${format(new Date(), 'dd.MM.yyyy')} - ${format(daysSinceEpochToDate(expirationDay), 'dd.MM.yyyy')}`
```
Comment (michael-markl) [inline administration/src/project-configs/nuernberg/config.ts:30]: I would prefer a `Gültig bis` only.

File: administration/src/project-configs/bayern/config.ts [hunk 2]
```diff
@@ -16,6 +19,17 @@ const config: ProjectConfig = {
...
     issuer: 'Bayerische Staatsministerium für Arbeit und Soziales, Familie und Integration',
+    infoToDetails: (info, region) => {
+      const expirationDay = info.expirationDay ?? 0
+      const expirationDate =
+        expirationDay > 0 ? format(daysSinceEpochToDate(expirationDay), 'dd.MM.yyyy') : 'unbegrenzt'
+
+      const cardType = info.extensions?.extensionBavariaCardType?.cardType
+      return `Name: ${info.fullName}
+Kartentyp: ${cardType == BavariaCardType.STANDARD ? 'Blau' : 'Gold'}
+Gültigkeit: ${format(new Date(), 'dd.MM.yyyy')} - ${expirationDate}
```
Comment (michael-markl) [inline administration/src/project-configs/bayern/config.ts:30]: I would prefer a Gültig bis only.

**Review by michael-markl — APPROVED** [review]
Looking goood.... Have you checked that there's enuogh space for the name on the physical card?

**Review by michael-markl — APPROVED** [review]

**Pull request merged**